Bump dependency of illuminate/contracts to allow Laravel 11

mylanconnolly opened the pull request:
Hello, I was using this library for my new Laravel project and as I was checking for the viability of updating to Laravel 11 I noticed that this project had a dependency on `illuminate/contracts` version 10.x. I allowed version 11.x in the dependencies and everything continues to seem to work correctly on my end.

I am not super experienced in the PHP ecosystem, however, so it's possible I'm not doing something correctly. If I'm not, I apologize.

Thanks for considering my contribution!

mylanconnolly committed:
Bump dependency of illuminate/contracts

mylanconnolly commented:
Hey @cheesegrits just checking; could please check this change out when convenient?

Thanks!

jcc5018 commented:
I agree, I am still learning how to use GitHub properly.  @mylanconnolly how can i install these packages with the |^11.0  part added? I tried requiring your fork, but that didnt work. I have 6 packages all related to filament with the same issue, and i cant run composer update with that dependency not defined

mylanconnolly commented:
@jcc5018 how I imported my fork into my project was:

Add a repository to my composer.json file like so:

```json
{
  "type": "vcs",
  "url": "https://github.com/mylanconnolly/filament-phone-numbers"
}
```

Ensure that the `require` line itself is like so: `"cheesegrits/filament-phone-numbers": "3.x-dev",`.

After I did that, I ran a `composer update` and it pulled my fork rather than the published version. Hope this helps!

mylanconnolly commented:
Hey @cheesegrits just wanted to check if you would be able to look this over. Thanks!

mylanconnolly committed:
Fix Unknown column 'phone' in 'where clause

This was caused by the column being hard-coded to 'phone'. I modified this to be the `name` attribute, instead.

mylanconnolly commented:
Just as an FYI I think I was able to solve #1 as well, and included the fix. Until it gets merged, my GitHub branch is available for anyone experiencing this issue.

Thanks!

Yo1L commented:
@cheesegrits can you accept these commits ?

imami commented:
@cheesegrits
